content/influxdb/cloud-dedicated/reference/client-libraries/v3/python.md

```diff
@@ -15,45 +15,35 @@ aliases:
 related:
   - /influxdb/cloud-dedicated/query-data/execute-queries/troubleshoot/
 list_code_example: >
-  <!-- Import for tests and hide from users.
+  
   ```python
-  import os
-  ```
-  -->
-  <!--pytest-codeblocks:cont-->
-  ```python
-  from influxdb_client_3 import(InfluxDBClient3,
-                                WriteOptions,
-                                write_client_options)
-
-  # Instantiate batch writing options for the client
-
-  write_options = WriteOptions()
-  wco = write_client_options(write_options=write_options)
-
-  # Instantiate an InfluxDB v3 client
-
-  with InfluxDBClient3(host=f"{{< influxdb/host >}}",
-                        database=f"DATABASE_NAME",
-                        token=f"DATABASE_TOKEN",
-                        write_client_options=wco) as client:
-
-    # Write data in batches
+  Example: Writing and querying data
+  
+  # The following example demonstrates how to write sensor data into influxDB 
+  
+  # and retrieve data from the last 90 days for analysis. 
+
+  
+  # Write sensor data in batches from a CSV file
     client.write_file(file='./data/home-sensor-data.csv', timestamp_column='time',
                       tag_columns=["room"])
 
-    # Execute a query and retrieve data formatted as a PyArrow Table
-
+  # Execute a query and retrieve data formatted as a PyArrow Table
     table = client.query(
       '''SELECT *
          FROM home
          WHERE time >= now() - INTERVAL '90 days'
          ORDER BY time''')
+    
+    # This script assumes the client object is correctly configured with your database name, token, and host URL. 
+    # After the script runs, the table variable contains the data formatted as a PyArrow table.
+
     ```
+    
 ---
 
 The InfluxDB v3 [`influxdb3-python` Python client library](https://github.com/InfluxCommunity/influxdb3-python)
-integrates {{% product-name %}} write and query operations with Python scripts and applications.
+integrates {{% product-name %}} write and query operations with Python scripts and applications. 
```

Comment:
```suggestion
integrates {{% product-name %}} write and query operations with Python scripts and applications.
```

Reply:
Thank you for the direction! For this request I am a tad confused. Can you please explain the a change in this that you would like me to make here?